internal/config/config.go

```diff
@@ -81,6 +81,14 @@ type MTLS struct {
 	TLSSecretNamespace string `mapstructure:"secretNamespace"`
 }
 
+func (m MTLS) SecretNamespace() string {
```

Comment:
why not setting it as:
 - `v.SetDefault("clusterName", "cluster")`

Or in helm charts directly?

Reply:
thanks yes i need to get rid of this, we decided that we are not going to use this default